packages/menu/src/styles.ts

```diff
@@ -297,7 +297,7 @@ export const disabledTextStyle: Record<Theme, string> = {
  */
 export const focusedMenuItemContainerStyle: Record<Theme, string> = {
   [Theme.Light]: css`
-    &:focus-visible {
+    &:focus {
```

Comment:
@brookeyalof why are switching to focus?

Reply:
so when you open the menu, the first item looks focused. We can switch back if you prefer.

Reply:
ooo, our current version only shows the focus ring when using the keyboard, so maybe we keep it that way.

Reply:
kk